doc/nrf/ug_nrf5340.rst

```diff
@@ -356,107 +356,112 @@ You must manually add a network core project to the application core project to
 
-1. Follow the instructions in :ref:`gs_programming_ses` and open the application core sample (for example, :ref:`peripheral_lbs`) in |SES|.
-   Use ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuapp_ns`` as the build target.
-#. Build the sample as described in :ref:`gs_programming_ses`.
+1. Follow the initial steps in :ref:`gs_programming_ses` to import the application core sample project in |SES| using ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuapp_ns`` as the build target.
+   For example, select the :ref:`peripheral_lbs` sample.
+#. Build the sample as described in :ref:`gs_programming_ses` in the step about building and programming the project.
    This creates both the application core image and the network core image.
-#. Select :guilabel:`File` > :guilabel:`New Project`.
-
-    .. figure:: images/ses_nrf5340_netcore_new_project.png
-       :alt: Create New Project menu
+   When the build completes, return to this guide.
+#. Add the network core project to the application core project:
 
-       Create New Project menu
+   a. Select :guilabel:`File` > :guilabel:`New Project`.
 
-#. Select :guilabel:`Add the project to the current solution`.
+      .. figure:: images/ses_nrf5340_netcore_new_project.png
+         :alt: Create New Project menu
 
-    .. figure:: images/ses_nrf5340_netcore_add_project.png
-       :alt: Adding a project target for programming the network core
+         Create New Project menu
 
-       Adding a project target for programming the network core
+   #. Select :guilabel:`Add the project to the current solution`.
 
-#. Select the project template, project name, and project location.
+      .. figure:: images/ses_nrf5340_netcore_add_project.png
+         :alt: Adding a project target for programming the network core
 
-   * :guilabel:`An externally built executable for Nordic Semiconductor nRF`:
-     This template allows you to specify the network core HEX file to be programmed.
-     The HEX file is created by the build system.
+         Adding a project target for programming the network core
 
-   * :guilabel:`Name`: Specify the name of the project as it will appear in SES.
-     This example uses ``hci_rpmsg_nrf5340_netcore``.
+   #. Select the project template, project name, and project location.
 
-   * :guilabel:`Location`: Specify the location of the project.
-     This must be the same location as the current project/build folder.
-     Click :guilabel:`Browse` to open a dialog where you can navigate to the current project/build folder and click :guilabel:`Select Folder`.
+      * :guilabel:`An externally built executable for Nordic Semiconductor nRF`:
+        This template allows you to specify the network core hexadecimal file to be programmed.
+        The hexadecimal file is created by the build system.
+      * :guilabel:`Name`: Specify the name of the project as it will appear in SES.
+        For example, the :ref:`peripheral_lbs` sample uses ``hci_rpmsg_nrf5340_netcore``.
```

Comment:
Isn't the name up to the user? Does it need to be specified to given value for a sample?

Reply:
I. e. does it e.g. depend on used net core image ?